metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/api/SraObjectsImporterThroughApiTest.java

```diff
@@ -156,6 +157,42 @@ public void importReferenceSequenceTranscriptome() throws Exception {
         assertEquals(1, referenceSequenceRepository.count());
     }
 
+    @Test
+    public void importReferenceSequenceGcfAssemblyAccessionWithoutApiKey() throws Exception {
```

Comment:
Common code across both methods can be pushed a separate method

Reply:
calling a separate function for the whole test is not so good I guess.